src/components/BreadCrumb/BreadCrumb.vue

```diff
@@ -29,6 +29,7 @@
               exact
               :to="item.to"
               :href="item.href"
+              :disabled="item.disabled"
```

Comment:
This could use another unit test, or at least acknowledgement that this new property exists (in unit tests or story).